docs/advanced-topics/dockstore-cli/dockstore-cli-faq.rst

```diff
@@ -0,0 +1,97 @@
+Dockstore CLI FAQ
+=================
+
+How does launching with Dockstore CLI compare with cwltool?
+-----------------------------------------------------------
+
+The Dockstore CLI has utilities to generate JSON parameter files from
+entries on Dockstore (``dockstore tool convert``).
+
+When launching tools, the Dockstore CLI makes it easy to specify entries
+from Dockstore. We can also provision input and output files using HTTP,
```

Comment:
"specify entries from Dockstore" -- may not be clear, but I can't think of a good alternative. Something about auto-download? But perhaps it's fine.

Reply:
I tried "from the Dockstore website", how's that?